docs/book/v2/migration/preparing-for-v3.md

```diff
@@ -1,10 +1,31 @@
 # Preparing for Version 3
 
-Version 3 will introduce a number of backwards incompatible changes.
+Version 3 will introduce a number of backward incompatible changes.
 This document is intended to help you prepare for these changes.
 
 ## Removed Features
```

Comment:
Nothing has been removed yet, so the outline of the document should take this into account, for example:

- Deprecations
- Upcoming Signature Changes / Upcoming Behaviour & Signature Changes
- Features That Will Be Removed

It must be clear what is being documented here.